frontend/src/services/fetch/fetchers/clientApplicationFetcher.ts

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+interface applicationStartResponse {
```

Comment:
there's already an `applicationStartResponse` interface in `applicationFetcher`. Can this be called something else?